Dockerfile

```diff
@@ -36,7 +39,10 @@ COPY --parents .yarn/ yarn.lock .yarnrc.yml **/package.json packages/bind-mount/
 #
 # If the following issue is ever addressed, we can use that instead:
 # https://github.com/yarnpkg/berry/issues/6339
-RUN yarn dlx node-gyp install && yarn install --immutable --inline-builds && yarn cache clean
+RUN yarn dlx node-gyp install && \
+    yarn install --immutable --inline-builds && \
+    yarn cache clean && \
+    make python-deps
```

Comment:
I really do not like combining these. This will force us to re-install Python deps whenever any `package.json` file changes. Let's leave Python dependency installation as a separate step that runs first.